src/components/tabs/_macro.njk

```diff
@@ -11,7 +11,7 @@
         {% if params.noInitialActiveTab %}data-no-initial-active-tab="true"{% endif %}
     >
         {% set titleTag = params.titleTag | default("h2") %}
-        <{{ titleTag }} class="ons-tabs__title ons-u-fs-r--b ons-u-mt-no">{{ params.title }}</{{ titleTag }}>
+        <{{ titleTag }} class="ons-tabs__title ons-u-fs-r--b ons-u-mt-no {% if params.titleTagClass %}{{params.titleTagClass}}{% endif %}">{{ params.title }}</{{ titleTag }}>
```

Comment:
```suggestion
        <{{ titleTag }} class="ons-tabs__title ons-u-fs-r--b ons-u-mt-no{% if params.titleTagClass %} {{ params.titleTagClass }}{% endif %}">{{ params.title }}</{{ titleTag }}>
```
This change fixes some styling and will not introduce spaces into the class list if there is no `titleTagClass`